agave_app/ViewToolbar.cpp

```diff
@@ -101,6 +101,16 @@ ViewToolbar::ViewToolbar(QWidget* parent)
   rightViewButton->setFocusPolicy(Qt::NoFocus);
   toolbarLayout->addWidget(rightViewButton);
 
+  toolbarLayout->addItem(new QSpacerItem(spacing, 0, QSizePolicy::Fixed, QSizePolicy::Expanding));
```

Comment:
It took me some time to learn about Qt Widgets and seems they are well used here!

Reply:
Yes, the whole user interface is built with them!